.wp-env.json

```diff
@@ -0,0 +1,4 @@
+{
+	"core": null,
+	"plugins": [ ".", "./tests/e2e/plugins/testing-blocks" ]
```

Comment:
Points the local environment to the Block Lab plugin, and a testing plugin, so they can run in the environment.